SwiftGit2/Repository.swift

```diff
@@ -328,12 +328,12 @@ final public class Repository {
 	/// name - The name of the remote.
 	///
 	/// Returns the remote if it exists, or an error.
-	public func remoteWithName(_ name: String) -> Result<Remote, NSError> {
+	public func remote(withName name: String) -> Result<Remote, NSError> {
```

Comment:
I prefer `remote(withName: "origin")` because of [the guideline that method call sites should read as grammatical English phrases.](https://swift.org/documentation/api-design-guidelines/#methods-and-functions-read-as-phrases)

This also feels more correct to me because it is the way Swift imports the similarly named Objective-C function:

```objc
- (nullable id)remoteWithName:(nonnull NSString *)name;
```
becomes
```swift
func remote(withName name: String) -> Any?
```